@navikt/core/css/darkside/typography.darkside.css

```
@@ -251,7 +251,6 @@
   margin: -1px !important;
   overflow: hidden !important;
   padding: 0 !important;
```

Don't "need" to add this, but this seems to be added by most modern visually-hidden CSS to
```suggestion
  padding: 0 !important;
  clip-path: inset(50%);
```